build.gradle

```diff
@@ -7,64 +7,57 @@ Base directory: $projectDir
 Running script ${relativePath(buildFile)}
 """
 
-apply plugin: 'idea'
-apply plugin: 'eclipse'
 apply plugin: 'groovy'
 apply plugin: 'maven'
-apply plugin: 'git'
```

Comment:
Not sure about git, but you should probably keep idea and eclipse for IDE users.